CHANGES.md

```diff
@@ -1,89 +1,148 @@
-# Dendrite 0.1.0 (2020-10-08)
+# Changelog
+
+## Dendrite 0.2.0 (2020-10-20)
+
+### Important
```

Comment:
I would emphasise that these are breaking changes e.g if you're running polylith.